source/scene/entity/behaviors/CameraAnimation.cpp

```diff
@@ -0,0 +1,94 @@
+#include "CameraAnimation.h"
+//#include "../Components.h"
+#include"../Entity.h"
+#include <cmath>
+
+typedef DirectX::SimpleMath::Vector3 Position;
+typedef DirectX::SimpleMath::Quaternion Orientation;
+AnimationBehavior::AnimationBehavior(Entity entity)
+    : _entity(entity)
+{
+}
+
+void AnimationBehavior::Initialize()
+{
+    // TODO: load animation here for camera
+    work = true;
+    tOut = _entity.GetComponent<CameraComponent>().current_Transform;//get current orientation and position
```

Comment:
Being more explicit with variable names makes the code a lot easier to read.
In this case would could rename `tOut` to `transform_out`.